### ISSUE TITLE
Clicking Show Conflicts crashes the app

### ISSUE THREAD

**Issue by LanceEddleman** [millerscout/Kenshi-Mod-Manager#5]
Is there a log file or anything i can give to help on this.

When i click:
Show conflicts
 or 
TestConflicts

it just crashes  the app

**Comment by millerscout** [comment]
Hello @LanceEddleman, currently i didn't write anything to log errors.

working on a new version (with better GUI)

Either way your problem seems be that the conflictChecker isn't configured.

on your config.json

add the line

``` json
  "ConflictAnalyzerPath": "Mod Conflict Manager.exe"
```

your config file should look like this:
``` javascript
{
  "gamePath": " {Your game path.} ",
  "SteamModsPath": " {YourSteam Folder if applicable} ",
  "SteamPageUrl": "https://steamcommunity.com/sharedfiles/filedetails/?id=",
  "NexusPageUrl": "https://www.nexusmods.com/kenshi/search/?gsearch=",
  "ConflictAnalyzerPath": "Mod Conflict Manager.exe"
}
```

**Comment by millerscout** [comment]
This issue was fixed on the newest release v1.11 
https://github.com/millerscout/Kenshi-Mod-Manager/releases
